mozilla/activity-stream
# Preparing the release of Activity Stream 1.1.7

Hi all,

We are now preparing the release of A-S 1.1.7. The add-on has been made available at [pre-release channel](https://moz-activity-streams-prerelease.s3.amazonaws.com/dist/latest.html). Since this release also bases the "Activity Stream Variant" in the shield study, it only adds a few UI enhancements as well as various bugfixes on top of A-S 1.1.6.

# Highlights (Changelog)
* [new] Added a more informative page to replace the "Hang tight" spinner
* [fix] Fixed the SQL binding overflow issue in the metadata store
* [fix] Fixed the exception error message template in the metadata store
* [fix] Fixed the bug that made the context menu not clickable

# Release checklist
- QA
- Product
- Engineering

Please use the `release/block` or `release/uplift` to label the bug report as well as other release related issues. Once you finish the review, please check off the corresponding item in the above checklist.

Thanks
